src/app/private/[uid]/chapter-leader/home/@joinChapter/page.tsx

```diff
@@ -0,0 +1,31 @@
+import { UserJoinRequest } from "@components/user";
```

Comment:
We don't need this page, since we will enforce that a user can only access a view that's specific to their role.